src/main/java/com/edmunds/rest/databricks/restclient/AbstractDatabricksRestClientImpl.java

```diff
@@ -128,6 +133,18 @@ protected HttpGet makeGetMethod(String path, Map<String, Object> data) {
     return method;
   }
 
+  protected HttpPatch makePatchMethod(String path, Map<String, Object> data)
```

Comment:
what is the patch method used for in databricks api?

Reply:
From what I can see, this is it @samshuster https://docs.databricks.com/dev-tools/api/latest/scim.html#update-user-by-id-patch

Reply:
got it thanks @javamonkey79